QModManager/Utility/IOUtilities.cs

```diff
@@ -16,10 +16,6 @@ internal static class IOUtilities
             "SNUnmanagedData",
             "Subnautica_Data",
             "_CommonRedist",
-            "Subnautica_Data/Mono",
-            "Subnautica_Data/Plugins",
-            "Subnautica_Data/Resources",
-            "Subnautica_Data/StreamingAssets",
```

Comment:
Do we really need to see contents of these folders in logs?

Reply:
Look 2 lines above. Those folders are already covered.

Reply:
I see. I had added the parent "Subnautica_Data" folder in the last update. 👍

Reply:
I know :)